OSDOCS-16850 2 - CQA for OVNK-1 (#111189)
openshift/openshift-docs: networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc

OVN-Kubernetes is the default networking solution for {product-title} and {sno} deployments.
====

> OVN-Kubernetes, which arose from the OVS project, uses many of the same constructs, such as open flow rules, to decide how packets travel through the network. For more information, see Additional resources.

Reviewer: ```suggestion
OVN-Kubernetes, which arose from the OVS project, uses many of the same constructs. For example, OpenFlow rules, to direct packets travel through the network. For more information, see the Additional resources.
```
hard to translate that crazy sentence; also personification and consistency with material above

Author: OVN-Kubernetes, which arose from the OVS project, uses many of the same constructs. For example, the plugin applies OpenFlow rules to direct packet routing through the network infrastructure. For more information, see Additional resources.

I think this keeps alignment with personification/consistency/anthropomorphism addressed. *fingers crossed*
